stratum/glue/logging.cc

```diff
@@ -57,6 +59,11 @@ void InitStratumLogging() {
   if (FLAGS_logtostderr) {
     LogToStderr();
   }
+
+  LOG(INFO) << "Stratum version " << kBuildScmRevision << " ("
+            << kBuildScmStatus << ")"
+            << " built at " << absl::FromTimeT(kBuildTimestamp) << " on host "
+            << kBuildHost << " by user " << kBuildUser << ".";
```

Comment:
Recent version of glog added `SetApplicationFingerprint`, check if it can replace this.